Makefile

```diff
@@ -10,3 +10,7 @@ run:
 	./make/tasks/run.sh $(ARGS)
 generate/cloud-openapi-code:
 	./make/tasks/generate/cloud-openapi-code.sh
+lint-fix:
+	./make/tasks/lint-fix.sh
+lint:
+	./make/tasks/lint.sh
```

Comment:
I would make `lint.sh` take a --fix parameter and handle the change there

Not sure if it's also possible to define arguments for make (I guess not)